front/src/app/page.tsx

```diff
@@ -13,9 +13,9 @@ export default async function Home({
     const logs = await LogsData.getLogs(searchParams);
 
     return (
-        <div className="flex flex-col gap-4 w-full">
+        <>
             <FilterHeader />
             {logs.length === 0 ? <EmptyLogs /> : <LogsTable logs={logs} />}
-        </div>
+        </>
```

Comment:
Maybe combine this into a file under `@/features/FilterListScreen.tsx`?

Reply:
I don't know if the query should be moved too, so I just extracted line 18 into `features/tables/TableBody`